dashboard/client/src/pages/serve/ServeApplicationDetailPage.tsx

```diff
@@ -0,0 +1,239 @@
+import {
+  Box,
+  createStyles,
+  InputAdornment,
+  makeStyles,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+  TextField,
+  TextFieldProps,
+  Typography,
+} from "@material-ui/core";
+import { Autocomplete, Pagination } from "@material-ui/lab";
+import dayjs from "dayjs";
+import React, { ReactElement } from "react";
+import { useParams } from "react-router-dom";
+import { CodeDialogButton } from "../../common/CodeDialogButton";
+import { CollapsibleSection } from "../../common/CollapsibleSection";
+import { DurationText } from "../../common/DurationText";
+import { MetadataSection } from "../../components/MetadataSection";
+import { StatusChip } from "../../components/StatusChip";
+import { HelpInfo } from "../../components/Tooltip";
+import { MainNavPageInfo } from "../layout/mainNavContext";
+import { useServeApplicationDetails } from "./hook/useServeApplications";
+import { ServeDeploymentRow } from "./ServeDeploymentRow";
+
+const useStyles = makeStyles((theme) =>
+  createStyles({
+    table: {
+      tableLayout: "fixed",
+    },
+    helpInfo: {
+      marginLeft: theme.spacing(1),
+    },
+  }),
+);
+
+const columns: { label: string; helpInfo?: ReactElement; width?: string }[] = [
+  { label: "Name" },
+  { label: "Status" },
+  { label: "Message", width: "30%" },
+  { label: "Replicas" },
```

Comment:
Is there a more specific term we could use? It's unclear if this means the target number of replicas or the current number of running replicas.

Based on the implementation, it looks like this is the target number of replicas set either by the user or by the autoscaling policy. How about `Target replicas`?

Reply:
Actually in the follow-up (once the replicas API is available) this will not represent target number of replicas but instead the active number of replicas. 

Do you think we should still rename the column in that case?

Reply:
Yeah, I think renaming is still a good idea. Serve developers manually deal with `num_replicas` which is the target number of replicas, so it's not necessarily obvious that this column refers to active replicas.

I'm okay with renaming it in the follow up though, since we'll change the behavior there. I'll resolve this comment for now.